configure.py

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env python3
 #
 # Copyright 2021-2022 GoPro Inc.
+# Copyright 2023 Nope Project
```

Comment:
Usually the most up-to-date goes on top (see for example in `android_imagereader.c`)

Reply:
Done.